README.md

```diff
@@ -42,6 +42,41 @@ Experimental implementation of the GPro and GRank measures can be found in the "
 
 ```java -jar amie3.jar -bias amie.mining.assistant.experimental.[GPro|GRank] [TSVFile]```
 
+### Use with remote knowledge base server
+
+Since loading knowledge graphs can take a significant amount of memory space and loading time, it is possible to run 
+AMIE with a remote autonomous knowledge base, slicing the architecture into two parts communicating over network.
+
+Bellow is a basic setup example to use AMIE with a remote knowledge base.
+
+#### Server-side
+
+```java -jar amie3.jar -server WS [TSVFile] -port <Server Port (default: 9092)>```
+
+#### Client-side
+
+```java -jar amie3.jar -client WS -serverAddress <Server Address (default: localhost:9092)>```
+
+#### Choosing protocol
+As of now, two communication protocols are available:
+- WebSocket, which is the recommended choice for using AMIE in remote KB mode. (```WS``` option value)
+- REST, which is only recommended for testing and development purposes (significantly slower). (```REST``` option value)
```

Comment:
Is there any advantage of REST over WebSocket (e.g., easier installation)? Otherwise I am afraid people will always use WebSocket.